src/SEBT.Portal.Api/Dockerfile

```diff
@@ -1,12 +1,26 @@
-﻿FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+﻿FROM mcr.microsoft.com/dotnet/aspnet:10.0 AS base
 USER $APP_UID
 WORKDIR /app
 EXPOSE 8080
 EXPOSE 8081
 
-FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:10.0 AS build
 ARG BUILD_CONFIGURATION=Release
+
+# Install Node.js and pnpm in the build stage
+RUN apt-get update && apt-get install -y curl
+RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash -
```

Comment:
Are we planning on using Node 20?  I think the latest LTS for Node is 24

Reply:
I confirmed with @adbergen that we're planning to use Node 24, so I would update this to get Node 24 with `APT`